library/private/animationManager.h

```diff
@@ -87,6 +87,18 @@ class animationManager
     return Playing;
   }
 
+  /**
+   * Set the interactor event loop time and compute the number of event loop needed to trigger an
+   * actual tick
+   * TODO rework this API ?
+   */
+  void SetInteractorEventLoopTime(double loopTime);
```

Comment:
What do you think of this API @Meakk ?

Reply:
Why not matching the animation tick rate to the actual event timestep?